java/pom.xml

```diff
@@ -1,7 +1,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>org.codingdojo</groupId>
+  <groupId>com.bacheconsulting</groupId>
```

Comment:
Actually, I would prefer all the code to be in the 'org.codingdojo' package the same as pretty much all my katas. Although to be fair I don't own that domain. 'org.sammancoaching' might be better.

Reply:
I changed both the package in Java files (and you seem to be OK with that) as well as Maven coordinates. I can revert this back to `org.codingdojo`, however there will be a little discrepancy between the two then.

And as you said, you technically don't own the domain, so anybody else could possibly use the same coordinates and cause a clash (which is why Java world uses _reverse domain notation_ to prevent these collisions). However, nobody is ever expected to publish this artifact to Maven, so we can accept the risk and go with `org.codingdojo` if you like.

Should I change the package in Java files too?

Reply:
yes please change all to org.codingdojo